sample/src/main/java/co/nimblehq/sample/ui/surveylist/SurveyListActivity.kt

```diff
@@ -0,0 +1,36 @@
+package co.nimblehq.sample.ui.surveylist
+
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import androidx.lifecycle.ViewModelProvider
+import co.nimblehq.sample.databinding.ActivitySurveyListBinding
+
+class SurveyListActivity : AppCompatActivity() {
+
+    private lateinit var surveyListViewModel: SurveyListViewModel
+    private lateinit var binding: ActivitySurveyListBinding
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        binding = ActivitySurveyListBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+        title = "Survey List"
+
+        surveyListViewModel = ViewModelProvider(this, SurveyListModelFactory())
+            .get(SurveyListViewModel::class.java)
+
+        surveyListViewModel.surveyListResult.observe(this@SurveyListActivity, {
+            it.success?.let { result ->
+                (binding.surveyListView.adapter as SurveyAdapter).updateData(result.listSurvey)
+            }
+
+        })
+        binding.surveyListView.adapter = SurveyAdapter()
+
+        surveyListViewModel.getSurveyList()
+
+    }
+
```

Comment:
avoid these redundant linebreak 🙏

Reply:
@sleepylee What is still here?

Reply:
<img width="793" alt="Screen Shot 2021-09-21 at 10 36 30 PM" src="https://user-images.githubusercontent.com/88996251/134201778-17c9f896-fd25-420a-9058-08d64ad16ab1.png">

Reply:
@sleepylee Okay, I got it. Let me double check with others. 🙏